deposit/src/main/java/edu/unc/lib/deposit/normalize/AbstractMETS2N3BagJob.java

```diff
@@ -99,7 +99,7 @@ protected void assignPIDs(Document mets) {
 				div.setAttribute("CONTENTIDS", sb.toString());
 			}
 			
-			PremisLogger premisLogger = getPremisLogger(pid);
```

Comment:
actually it looks like you did update getPremisLogger, so I'd say to just use that and roll back the changes to AbstractMETS2N3BagJob and CDRMETS2N3BagJob